docs/docs/recipes/integrate-logto/android.mdx

```diff
@@ -49,12 +49,10 @@ dependencies {
 
 ## Configure
 
-### Configure Redirect URI
-
-The `Redirect URI` indicates the endpoint that the application used to receive authorization results, which is implemented by the Logto Android SDK internally.
-
 ### Configure Redirect URI in Admin Console
```

Comment:
you need put an admonition here like other recipes

Reply:
updated.

Reply:
well i was think about the import of `<AppNote />` and `<SignInNote />` etc. would you like to go through the code and follow the latest integration guide format?

Reply:
updated.